pkg/plugins/common/kustomize/v1/scaffolds/init.go

```diff
@@ -62,23 +62,44 @@ func (s *initScaffolder) Scaffold() error {
 		machinery.WithConfig(s.config),
 	)
 
```

Comment:
@laxmikantbpandhare @SayakMukhopadhyay 

It shows that we can also remove 

```
generatorOptions:
  disableNameSuffixHash: true
```

when the flag is not informed because it shows to be used only in the generate option for configComponent

Reply:
Yes, Modified the` kustomization.yml` file for both `v1` and `v2-alpha`.